src/pudl/metadata/fields.py

```diff
@@ -963,6 +967,11 @@
         "type": "boolean",
         "description": "Can the generator burn multiple fuels?",
     },
+    "mwh_sold": {
+        "type": "number",
+        "description": "Megawatt hours sold.",
+        "unit": "MWh",
+    },
```

Comment:
I'd suggest using `sales_mwh` instead here which is already defined. I think this is currently only being used for EIA, but it seems analogous.

Reply:
Changed